openshift/release/release-notes.md

```diff
@@ -0,0 +1,279 @@
+# OpenShift Pipelines TP-1.3
+
+# Tekton Pipelines v0.19.0
+
+## Features
+
+* :sparkles: Do not allow use of deprecated Conditions with custom tasks ([#3601](https://github.com/tektoncd/pipeline/pull/3601))
+
+  Disallow use of Conditions in pipeline tasks that reference custom tasks
+
+* :sparkles: Add variable expansion for ImagePullPolicy in Tasks. See #3423 ([#3488](https://github.com/tektoncd/pipeline/pull/3488))
+
+  Add variable expansion in Tasks for fields:
+
+  * spec.steps[].imagePullPolicy
+  * spec.sidecar[].imagePullPolicy
+
+* :sparkles: Add an unsuccessful TaskRun testcase in conformance tests ([#3454](https://github.com/tektoncd/pipeline/pull/3454))
+
+## Deprecation Notices
+* :rotating_light: The PascalCase fields in WhenExpressions is deprecated
+  
+  Tasks and Pipelines with WhenExpressions that were created using Tekton Pipelines v0.16.x need to be reapplied to fix the case of their json annotations ([#3570](https://github.com/tektoncd/pipeline/pull/3570))
+
+## Fixes
+* :bug: Fix PipelineRun serviceAccountNames for finally tasks ([#3560](https://github.com/tektoncd/pipeline/pull/3560))
+
+  Fixes a bug where PipelineRun's serviceAccountNames and taskPodSpecs couldn't be applied on finally tasks and resulted in an error.
+
+* :bug: Fix duplicate TaskRuns when Pipeline/PipelineRun labels are changed ([#3558](https://github.com/tektoncd/pipeline/pull/3558))
+
+  Fix an issue where the PipelineRun controller could create duplicate TaskRuns if a Pipeline's or PipelineRun's labels are changed while the PipelineRun is running
+
+* :bug: Consider not-found pod as permanent error when taskrun is done ([#3542](https://github.com/tektoncd/pipeline/pull/3542))
+
+  Fix an issue where the taskrun controller would continue reconciling completed taskruns if pods in case of evicted pods
+
+* :bug: Fix script step on CRI-O runtime ([#3526](https://github.com/tektoncd/pipeline/pull/3526))
+## Misc
```

Comment:
We do not document items in Misc, section too, so IMHO can be removed, unless you think this is really relevant for downstream/end users.

Reply:
Removed